### PR TITLE
Add revocation examples

### PR DESCRIPTION
**Pull request by abdulmth**
# Description of change
Add example for revoking verifiable credentials in Rust and Wasm.

## Links to any relevant issues
#1060 

## Type of change
Add an `x` to the boxes that are relevant to your changes.

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] Enhancement (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Fix

### CODE REVIEW

**Comment by eike-hass** [comment]
I assume for Wasm the example needs to be added to the tests as well. For Rust that is automatic, right?

File: bindings/wasm/examples/src/0_basic/7_revoke_vc.ts [hunk 1]
```diff
@@ -0,0 +1,157 @@
...
+    });
+
+    // Sign Credential.
+    let signedVc = issuerDocument.signCredential(unsignedVc, keypairIssuer.private(), "#key-1", ProofOptions.default());
+    console.log(`Credential JSON > ${JSON.stringify(signedVc, null, 2)}`);
+
+    // ===========================================================================
+    // Revocation of the Verifiable Credential.
+    // ===========================================================================
+
```
Comment (olivereanderson) [inline bindings/wasm/examples/src/0_basic/7_revoke_vc.ts:105]: Perhaps we should also demonstrate that the credential can be verified successfully prior to revocation?

File: examples/0_basic/7_revoke_vc.rs [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+    .build()?;
+
+  // Sign the Credential with the issuer's verification method.
+  issuer_document.sign_data(&mut credential, key_pair.private(), "#key-1", ProofOptions::default())?;
+  println!("Credential JSON > {:#}", credential);
+
+  // ===========================================================================
+  // Revocation of the Verifiable Credential.
+  // ===========================================================================
+
```
Comment (olivereanderson) [inline examples/0_basic/7_revoke_vc.rs:133]: Again maybe we should demonstrate that the credential can be successfully verified at this point before it gets revoked?

File: examples/0_basic/7_revoke_vc.rs [hunk 3]
```diff
@@ -0,0 +1,192 @@
...
+  let original_method: DIDUrl<IotaDID> = issuer_document.resolve_method("#key-1", None).unwrap().id().clone();
+  issuer_document.remove_method(&original_method).unwrap();
+
+  // Publish the changes.
+  let alias_output: AliasOutput = client.update_did_output(issuer_document.clone()).await?;
+  let alias_output: AliasOutput = AliasOutputBuilder::from(&alias_output).finish(client.get_token_supply()?)?;
+  client.publish_did_output(&secret_manager_issuer, alias_output).await?;
+
+  // We expect the verifiable credential to be revoked.
+  let mut resolver: Resolver = Resolver::new();
```
Comment (olivereanderson) [inline examples/0_basic/7_revoke_vc.rs:174]: nit: Possibly better to use `Resolver<IotaDocument>` here instead. In which case line 176 also needs to be changed to use an `IotaDocument` instead. 

You can decide what you want to do here.

**Review by olivereanderson — APPROVED** [review]
LGTM!

Wrote a few comments.

**Pull request merged**